### ISSUE TITLE
add formats?

### ISSUE THREAD

**Issue by cholmes** [cportele/ogcapi-building-blocks#5]
Should we have a 'formats' section where we add in geopackage, geojson, wkt, wkb, geotiff/cog, zarr, etc? Or does that belong in its own repo of like 'format building blocks'? It feels to me like the other bit an implementor new to geospatial would need in their API. Maybe it's just a page of documentation that guides people through what format to use.

**Comment by cportele** [comment]
Agreed. Eventually this needs to be included, for now a descriptive page that provides and overview and links could be a good start.
